hack/ci/lvm-setup.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
Why is this needed?

Reply:
This is just a WIP PR, Will need to do some changes @balamurugana